<ui version="4.0" >
 <class>CategoryInfoDialogClass</class>
 <widget class="QDialog" name="CategoryInfoDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CategoryInfoDialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="labelCategoryCode" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Category code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSpeed" >
       <property name="text" >
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAdditionalRemark" >
       <property name="text" >
        <string>Additional remark</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLineEdit" name="lineEditCategoryCode" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSpeed" />
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAdditionalRemark" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="checkBoxDrivingLicenseRequired" >
     <property name="text" >
      <string>Driving License required</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="rightMargin" >
      <number>11</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLineEdit" name="lineEditStatusBar" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>lineEditCategoryCode</tabstop>
  <tabstop>lineEditSpeed</tabstop>
  <tabstop>lineEditAdditionalRemark</tabstop>
  <tabstop>checkBoxDrivingLicenseRequired</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>lineEditStatusBar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>CategoryInfoDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>150</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel" >
     <x>174</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
